springcloud/src/main/java/org/axonframework/extensions/springcloud/commandhandling/SpringCloudCommandRouter.java

```diff
@@ -321,9 +324,10 @@ private Member buildLocalMember(ServiceInstance localServiceInstance) {
     }
 
     private Member buildRemoteMember(ServiceInstance remoteServiceInstance) {
-        URI remoteServiceUri = remoteServiceInstance.getUri();
-        return new SimpleMember<>(buildSimpleMemberName(remoteServiceInstance.getServiceId(), remoteServiceUri),
-                                  remoteServiceUri,
+		URI serviceWithContextRootUri = buildRemoteUriWithContextRoot(remoteServiceInstance);
```

Comment:
Indenting in this method seemed to have moved one tab, which I don't think is necessary.